blackjack/api/views.py

```diff
@@ -20,8 +20,17 @@ def __init__(self):
         random.shuffle(self.deck_cards)
 
 
+@api_view(["GET", "POST", "PUT"])
+def deal(request, id):
+    data = json.loads(request.body)
+    username = data['username']
+    deck = data['deck']
+    player_hand = data['player_hand']
+    dealer_hand = data['dealer-hand']
 
+    # how do I get this into the hand that called the function?
+    card_dealt = Game.deck_cards.pop()
```

Comment:
dont forget to create an instance of the `Game` class! don't want to use the class directly

Reply:
I remember that now, I think I fixed it